pkg/costmodel/costmodel.go

```diff
@@ -867,6 +867,11 @@ func (cm *CostModel) GetNodeCost() (map[string]*costAnalyzerCloud.Node, error) {
 	for _, n := range nodeList {
 		name := n.Name
 		nodeLabels := n.Labels
+		if nodeLabels == nil {
+			log.Warnf("Got node '%s' with no labels", name)
```

Comment:
```suggestion
			log.Debugf("GetNodeCost: Found node '%s' with no labels", name)
```

Just to keep logs clean, proposing we keep this at Debug? We can surface to Warn/Error if we believe there are downstream consequences of no labels?